src/app/communities/communities-new.component.html

```diff
@@ -6,6 +6,22 @@
         <input matInput id="communities-new-title" placeholder="Title" formControlName="title" [required]="true">
       </mat-form-field>
     </div>
+    <div fxFlex>
+      <span>Community type:</span>
+      <mat-button-toggle-group formControlName="type" class="communities-new-type">
+        <mat-button-toggle value="OPENED" class="communities-new-type-option">Opened</mat-button-toggle>
+        <mat-button-toggle value="CLOSED" class="communities-new-type-option">Closed</mat-button-toggle>
+      </mat-button-toggle-group>
+      <div *ngIf="communityForm.get('type').value === 'CLOSED'" class="communities-new-type-hint">
+        <mat-hint>
+          You have chosen a closed community. Since you are the community-manager of this community, you will have the
```

Comment:
In my point of view a community manager should be prompted with a modal dialog where she / he should press a confirmation button letting the system know he read the information and accepts it. Otherwise there is no interaction with a user and the system cannot be sure she / he saw the information message.

Reply:
was moved to the modal dialog